src/main/java/org/jvnet/hudson/plugins/shelveproject/ShelveProjectExecutable.java

```diff
@@ -86,7 +84,8 @@ private void wipeoutWorkspace()
         }
 
     }
-    
+
+    @SuppressFBWarnings("RV_RETURN_VALUE_IGNORED_BAD_PRACTICE")
```

Comment:
It would be better to fix such issues by switching to the new `Files` API available in Java 7 and above. This is a real defect in the code